api/clients/v2/payloaddispersal/payload_disperser_config.go

```diff
@@ -0,0 +1,63 @@
+package payloaddispersal
+
+import (
+	"time"
+
+	"github.com/Layr-Labs/eigenda/api/clients/v2"
+)
+
+// PayloadDisperserConfig contains an embedded PayloadClientConfig, plus all additional configuration values needed
+// by a PayloadDisperser
+type PayloadDisperserConfig struct {
+	clients.PayloadClientConfig
+
+	// DisperseBlobTimeout is the duration after which the PayloadDisperser will time out, when trying to disperse a
+	// blob
+	DisperseBlobTimeout time.Duration
+
+	// BlobCertifiedTimeout is the duration after which the PayloadDisperser will time out, while polling
+	// the disperser for blob status, waiting for BlobStatus_CERTIFIED
+	BlobCertifiedTimeout time.Duration
+
+	// BlobStatusPollInterval is the tick rate for the PayloadDisperser to use, while polling the disperser with
+	// GetBlobStatus.
+	BlobStatusPollInterval time.Duration
+
+	// The timeout duration for contract calls
+	ContractCallTimeout time.Duration
+}
+
+// GetDefaultPayloadDisperserConfig creates a PayloadDisperserConfig with default values
+func GetDefaultPayloadDisperserConfig() *PayloadDisperserConfig {
+	return &PayloadDisperserConfig{
```

Comment:
if this is only used in checkAndSetDefaults below, then prob can keep private?

Reply:
[f48dea34](https://github.com/Layr-Labs/eigenda/pull/1368/commits/f48dea34ea9e3a1b0364c347e2e1f9306a84420b)